sdk/avalon_sdk/tcf_connector.toml

```diff
@@ -59,8 +59,8 @@ solc_version = "v0.5.15"
 #Provider for test network. The default url is for a Hyperledger Besu client.
 #This could be replaced with a Ropsten(Infura as the IAAS) provider or a Ganache client.
 #provider = "http://10.66.245.80:8545"
-provider = "http://10.66.245.80:22001"
-event_provider = "http://10.66.245.80:22011"
+provider = "http://172.13.0.3:8555" 
```

Comment:
Any corresponding change required in truffle-config.js for besu. The avalon network section uses port 22001